bool success_prev # did the previous action succeed?
string uuid_prev # prev action UUID
int8 action_type_prev # prev action_type

# the previous action can return details
string action_details # json as string

---
bool success # is able to provide next action
string uuid
int8 action_type
string action_block # json as string

